_inc/client/traffic/search.jsx

```diff
@@ -32,30 +35,32 @@ class Search extends React.Component {
 					hasChild
 					module={ { module: 'search' } }
 					support={ {
-						text: __( 'Replaces the default WordPress search with a faster, filterable search experience.' ),
+						text: __( 'Jetpack Search supports many customizations.' ),
 						link: 'https://jetpack.com/support/search',
```

Comment:
Should we link to the customization support page here? https://jetpack.com/support/search/customize-search/